scripts/tests/happy_test_wrapper.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/env python
+
+# Copyright (c) 2020 Project CHIP Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This is a wrapper for running happy tests.
+
+import os
+import sys
+import argparse
+
+CHIP_PATH = os.path.realpath(os.path.join(
+    os.path.dirname(__file__), "../.."))
+HAPPY_TEST_PATH = os.path.join(CHIP_PATH, "src/test_driver/happy/tests")
+
+test_environ = os.environ.copy()
+
+parser = argparse.ArgumentParser(description='Warpper to run happy tests')
+parser.add_argument('test_script', type=str,
+                    help='The name of test')
+parser.add_argument('--test-bin-dir', dest='bin_dir', type=str, nargs='?', default='.',
+                    help='The path of test binaries')
+parser.add_argument('--ci', dest='ci', type=bool, nargs='?', default=False,
+                    help='Set this if running script under venv but happy is installed globally')
+
+if __name__ == '__main__':
+    args = parser.parse_args()
+    print(args, file=sys.stderr)
+    print(test_environ, file=sys.stderr)
+    # GN will run Python in venv, which will break happy test
```

Comment:
happy is not installed in venv

Reply:
we can install it there?

Reply:
happy can be run in virtualenv, there may be one bug in recent happy change.

Reply:
Did some attempts to get happy available in venv, but failed, one of the reason is failed to install psutil in venv, see https://github.com/erjiaqing/connectedhomeip/runs/1174544304 for details. I think this can be a follow up.